app/channels/publish_channel.rb

```diff
@@ -0,0 +1,15 @@
+class PublishChannel < ApplicationCable::Channel
+  def subscribed
+    puts("[PublishChannel#subscibed] #{params.inspect}")
+    stream_for params["channelId"]
+    transmit(consumerChannelId: Time.now.to_i)
```

Comment:
I know you are planning on preserving this value at some point, I would request this change to being initialized as a random value at that point

Reply:
Oh, I totally agree -- just want a test value here.  Definitely will be properly random later.